### ISSUE TITLE
Completion doesn't work

### ISSUE THREAD

**Issue by vkz** [purescript-emacs/emacs-psci#8]
Latest build of `purescript-mode` from Github and psci-20150328.1201 installed from Melpa. TAB completion doesn't show loaded symbols, say trying to complete `Chap` doesn't offer `Chapter2` even though:

``` shell
> :s loaded
Chapter2
Control.Monad.Eff
Control.Monad.Eff.Unsafe
Control.Monad.ST
Data.Function
Debug.Trace
Math
Prelude
Prelude.Unsafe
```

Maybe it needs `completion-at-point-functions` properly setup? Like sending token + "\t" to psci process to get candidates? I think that's what `nodejs-repl` does. 

I'm on:

`````` shell
"GNU Emacs 24.5.1 (x86_64-apple-darwin14.3.0, NS apple-appkit-1347.57)
 of 2015-06-09 on 95.108.174.109-red.dhcp.yndx.net"```
``````

**Comment by ardumont** [comment]
Hello,

> Latest build of purescript-mode from Github and psci-20150328.1201 installed from Melpa. TAB completion doesn't show loaded symbols, say trying to complete Chap doesn't offer Chapter2 even though:
> 
> > :s loaded
> Chapter2
> Control.Monad.Eff
> Control.Monad.Eff.Unsafe
> Control.Monad.ST
> Data.Function
> Debug.Trace
> Math
> Prelude
> Prelude.Unsafe
> 
> Maybe it needs completion-at-point-functions properly setup? Like sending token + "\t" to psci process to get candidates? I think that's what nodejs-repl does.

I do not remember having tested completion.
But surely, it must be implemented (or fixed).

> 
> I'm on:
> 
> "GNU Emacs 24.5.1 (x86_64-apple-darwin14.3.0, NS apple-appkit-1347.57)
>  of 2015-06-09 on 95.108.174.109-red.dhcp.yndx.net"```

Anyway, thanks for your feedback.
I am under water right now.
If you have some time to propose a PR, I'll merge it.

Otherwise, I'll try to have a look as soon as it's possible.

Cheers,

**Comment by kritzcreek** [comment]
Fixed in #27
